### PR TITLE
Amls 4746 - phase-2-toggle removal

### PR DESCRIPTION
**Pull request by samzaroughmrc**
removed code that is not supported by phase-2-removal

## Checklist

- [x] Breaking change (fix or feature that would cause existing functionality to change).
- [x] Build passing.
- [x] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [x] Requires acceptance test run.
- [x] Appropriate labels added.
- [x] RELEASE NOTES ADDED.

### CODE REVIEW

File: app/models/des/responsiblepeople/ResponsiblePersons.scala [hunk 1]
```diff
@@ -131,25 +131,15 @@ object ResponsiblePersons {
     val (training, trainingDesc) = convTraining(rp.training)
     val (expTraining, expTrainingDesc) = convExpTraining(rp.experienceTraining)
 
-    val msbOrTcsp: Option[MsbOrTcsp] = if (AmlsConfig.phase2Changes) {
-      None
-    } else {
-      rp.approvalFlags.hasAlreadyPassedFitAndProper.fold[Option[MsbOrTcsp]](None) { x => Some(MsbOrTcsp(x)) }
-    }
+    val msbOrTcsp: Option[MsbOrTcsp] = None
```
Comment (slavvco) [inline app/models/des/responsiblepeople/ResponsiblePersons.scala]: do we need to keep this?

File: app/models/des/responsiblepeople/ResponsiblePersons.scala [hunk 2]
```diff
@@ -131,25 +131,15 @@ object ResponsiblePersons {
...
 
-    val passedFitAndProperTest: Option[Boolean] = if (AmlsConfig.phase2Changes) {
+    val passedFitAndProperTest: Option[Boolean] =
       // TODO: When toggle is removed then this can be made non optional
       rp.approvalFlags.hasAlreadyPassedFitAndProper orElse Some(false)
```
Comment (slavvco) [inline app/models/des/responsiblepeople/ResponsiblePersons.scala]: what about this todo?

Comment (samzaroughmrc) [reply app/models/des/responsiblepeople/ResponsiblePersons.scala]: I already wondered, but it seems to bring other changes that are not related to removing the toggle removal. I would rather this done on a separate piece of work

File: app/models/des/responsiblepeople/ResponsiblePersons.scala (hunk 1 above)
Comment (samzaroughmrc) [reply app/models/des/responsiblepeople/ResponsiblePersons.scala]: same as reply below

File: app/models/des/responsiblepeople/ResponsiblePersons.scala (hunk 2 above)
Comment (slavvco) [reply app/models/des/responsiblepeople/ResponsiblePersons.scala]: ok then it can go to test

**Pull request merged**